documentation/content/en/books/handbook/x11/_index.adoc

```diff
@@ -53,7 +53,7 @@ endif::[]
 
 An installation of FreeBSD using bsdinstall does not automatically install a graphical user interface.
 This chapter describes how to install and configure Xorg, which provides the open source X Window System used to provide a graphical environment.
-It then describes how to find and install a desktop environment or window manager.
+It then describes how to customize the configuration and manage graphics drivers.
```

Comment:
Nice! Graphics drivers an important subsection in this chapter missing from the synopsis.

I noticed that customizing the configuration is mentioned in the previous line.
How do you feel about removing this whole line, and mentioning managing graphics drivers in the "you will know:" block?

Also XDM has been moved and shouldn't be in there.zapping